sdk/storage/azure-storage-blob/tests/test_block_blob.py

```diff
@@ -27,12 +27,14 @@
 
 from devtools_testutils import recorded_by_proxy
 from devtools_testutils.storage import StorageRecordedTestCase
+from .fake_credentials import CPK_KEY_HASH, CPK_KEY_VALUE
```

Comment:
Pipelines suggest that you'll need to remove the `.`s at the start of import statements
```suggestion
from fake_credentials import CPK_KEY_HASH, CPK_KEY_VALUE
```

Reply:
Oops, thanks. Fixed!